Dockerfile

```diff
@@ -1,52 +1,39 @@
-## semgrep build
-
-FROM python:3.7.7-alpine3.11 as build-semgrep
-RUN apk add --no-cache git
-COPY .git /home/pythonbuild/.git
-COPY semgrep /home/pythonbuild/semgrep/
-WORKDIR /home/pythonbuild/semgrep
-# downgrade to 20.1 since 20.1.1 reverts building in place
-# https://github.com/pypa/pip/issues/7555
-RUN pip install pip==20.1
```

Comment:
Need to keep this pinned https://github.com/docker-library/python/blob/7dc395a6b9fabce8685009dc385fa98f8a944d5c/3.7/alpine3.11/Dockerfile#L148 base image changed the pip version it uses

Reply:
No longer needed cause of comment above re: no longer needing .git directory

Reply:
We shouldn't need to pin the version anymore since we're no longer using that functionality to burn in SCM information.